src/app/modules/products/components/product-list/product-list.component.ts

```diff
@@ -11,13 +11,10 @@ import { Product } from "@shared/models/classes/product.class";
     styleUrls: ["./product-list.component.scss"]
 })
 export class ProductListComponent implements OnInit {
-    featuredProducts$: Observable<Product[]> = new Observable();
-    products$: Observable<Product[]> = new Observable();
+    @Input() products$: Observable<Product[]> = new Observable();
```

Comment:
no need for an observable, this should be:

@Input() products: Product[] = [];